tests/foreman/cli/test_webhook.py

```diff
@@ -0,0 +1,240 @@
+"""Test class for Webhook CLI
+
+:Requirement: Webhook
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Webhooks
+
+:Assignee: gsulliva
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from functools import partial
+from random import choice
+
+import pytest
+from box import Box
+from fauxfactory import gen_alphanumeric
+
+from robottelo.cli.base import CLIError
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.webhook import Webhook
+from robottelo.constants import WEBHOOK_EVENTS
+from robottelo.constants import WEBHOOK_METHODS
+from robottelo.datafactory import parametrized
+
+
+@pytest.fixture(scope='function')
+def webhook_factory(class_org, class_location):
+    def _create_webhook(org, loc, options=None):
+        """Makes a new Webhook and assets it's success"""
```

Comment:
maybe you meant "asserts" ...though I don't see any assertions here, maybe that was offloaded to `assert_created`?